floris/simulation/__init__.py

```diff
@@ -37,7 +37,22 @@
 import floris.logging_manager
 
 from .base import BaseClass, BaseModel, State
-from .turbine import average_velocity, axial_induction, Ct, power, rotor_effective_velocity, Turbine
+from .turbine import (
+    _filter_convert,
```

Comment:
Generally, a "private" function (the ones with leading underscore) shouldn't be exposed in the the public facing api for the package. Either rename the function to not have an underscore or change the calling code to get it from the module where it is implemented.

Reply:
Removed from `__init__.py`.